transformer_engine/pytorch/tensor/storage/mxfp8_tensor_storage.py

```diff
@@ -49,8 +49,8 @@ def backward(
         return grad, None
 
 
-class MXFP8TensorBase(QuantizedTensorBase):
-    """Mixin class that holds data attributes of MXFP8Tensor.
+class MXFP8TensorStorage(QuantizedTensorStorage):
+    """Mixin/storage class that holds data attributes of MXFP8Tensor.
```

Comment:
Nit: mix-in classes and storage classes are different things, so "mixin/storage class" doesn't mean much to me.
```suggestion
    """Mixin class that holds data attributes of MXFP8Tensor.
```